pkg/cosign/verify.go

```diff
@@ -684,27 +687,26 @@ func verifyInternal(ctx context.Context, sig oci.Signature, h v1.Hash,
 			return bundleVerified, err
 		}
 	}
-	// TODO: Verify TSA only and return the result with the tsa bundle
 	if co.TSAClient != nil {
 		bundleVerified, err = VerifyTSABundle(ctx, sig, co.TSAClient, co.TSACertChainPath)
 		if err != nil {
 			return false, fmt.Errorf("unable to verify TSA bundle: %w", err)
 		}
-
-		return bundleVerified, nil
 	}
 
-	bundleVerified, err = VerifyBundle(ctx, sig, co.RekorClient)
-	if err != nil {
-		return false, fmt.Errorf("error verifying bundle: %w", err)
+	if !co.SkipTlogVerify {
```

Comment:
nit: beyond this point everything is about tlog verification, so i think we can just skip it all by doing something like

```
if co.SkipTlogVerify {
  return true, nil
}
```